### ISSUE TITLE
Font Manager Font List Radio Selector Not Accessible

### ISSUE THREAD

**Issue by jakejackson1** [GravityPDF/gravity-pdf#1195]
**Description**
These inputs are not accessible via keyboard navigation, and it doesn't include an ARIA description about what it does.

**Step To Reproduce**
1. Go to Settings -> PDFs
2. Click on Advanced button next to the font selector
3. Tab through the List

**Expected behavior**
To be able to tab through each font in the list and mark it as selected. Should also receive screenreader output on the radio field selector explaining what it is. 

**Screenshots**
<img width="553" alt="Screen Shot 2021-03-01 at 2 55 57 pm" src="https://user-images.githubusercontent.com/2918419/109450120-453d5180-7a9e-11eb-80a0-38c6ca213473.png">
